packages/next-templates/src/app/wmebv/productenpagina-klachtenformulier/page.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+
+import {
+  UtrechtArticle,
+  UtrechtButton,
+  UtrechtButtonGroup,
+  UtrechtDigidLogo,
+  UtrechtHeading1,
+  UtrechtHeading2,
+  UtrechtIcon,
+  UtrechtLink,
+  UtrechtPage,
+  UtrechtPageContent,
+  UtrechtParagraph,
```

Comment:
Jij gebruikt de web component library en de component-library-react voor alleen sommige. Ik heb dat geswitcht zodat het consistent blijft of maakt dat niet uit?

Reply:
Dit is van een oudere commit (helemaal aan het begin heb ik de code van andere pagina gekopieerd om te testen of localhost dit liet zien). Bij het zelf bouwen van de pagina heb ik @utrecht/component-library-react gebruikt! 😊